src/nav/navSelectors.js

```diff
@@ -83,12 +84,19 @@ export const getInitialNavState = createSelector(
       !nav || (nav.routes.length === 1 && nav.routes[0].routeName === 'loading')
         ? getStateForRoute('main')
         : nav;
-
-    if (!config.startup.notification) {
+    const {
+      startup: { notification },
+    } = config;
+    if (!notification) {
       return state;
     }
 
-    const narrow = getNarrowFromNotificationData(config.startup.notification, usersById);
+    const narrow = getNarrowFromNotificationData(notification, usersById);
```

Comment:
Selectors should be pure functions.
Sending a notification from inside a selector is not a good idea!